app/views/suggest_a_register/complete.haml

```diff
@@ -11,19 +11,17 @@
 .govuk-width-container
   %main{class: 'govuk-main-wrapper govuk-!-padding-top-7 govuk-!-padding-bottom-8', role:'main'}
     .govuk-grid-row
-      .govuk-grid-column-full
-        %h1{class: 'govuk-heading-l govuk-!-margin-bottom-3'} Suggest a new register
       .govuk-grid-column-two-thirds
+        %h1{class: 'govuk-heading-l govuk-!-margin-bottom-3'} Suggest a new register
+
         %p You need to complete our engagement form to suggest a new register.
         %p The details you provide will help us to better understand your needs for a register, the data you have to offer and whether it is suitable to become a register.
-        %p.govuk-inset-text
-          = link_to 'Use our Google form', 'https://docs.google.com/forms/d/e/1FAIpQLSfw54dI5uYuF-15zH6Iu7KRQnj8lRY3Wcl9JI5hC80cENx-jw/viewform', {data: {'ga-label': 'Suggest a register Google form'}}
-          &nbsp;to suggest a new register.
+
+        %p.govuk-inset-text #{link_to 'Use our Google form', 'https://goo.gl/forms/GFaJgfnn7dH7AH5f1', {data: {'ga-label': 'Suggest a register Google form'}} } to suggest a new register.
```

Comment:
We may as well link to the full URL https://docs.google.com/forms/d/e/1FAIpQLScRNr-1oAxlt_fwy9JPEjzglnyi0qJVFKNw1odbAHX5ZV9fIQ/viewform to save a redirect.

Reply:
Sure thing - I left it as a shortened link as I (wrongly) assumed that the it had some bonus tracking in it.